scripts/docker_push.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# Dockerhub login
+[ ! -z "$DOCKER_PASSWORD" ] || [ ! -z "$DOCKER_USERNAME" ];
+  echo "DOCKER_USERNAME and DOCKER_PASSWORD must be set as environment variables!";
+  exit 1;
```

Comment:
In general you don't need to use `;` unless you want to run multiple commands in one go (useful for `if condition; then`, among other things), for case statements, and for command grouping with `{ cmd1; cmd2; ... cmdn; }`.